data/icon-overlay.js

```diff
@@ -187,3 +188,9 @@ function evNoop(ev) {
   ev.preventDefault();
   ev.stopImmediatePropagation();
 }
+
+function closeFullscreen() {
+  if(document.mozFullScreenEnabled){
```

Comment:
Please update this line to follow code conventions throughout the rest of the project.
* space after `if` statement
* space before curly bracket
```js
if (document.mozFullScreenEnabled) {
```
We should add this to our `eslintrc` so it doesn't show down our review process.